packages/components/select/src/useSelect.ts

```diff
@@ -783,13 +790,6 @@ export const useSelect = (props: ISelectProps, emit) => {
     width: `${Math.max(states.calculatorWidth, MINIMUM_INPUT_WIDTH)}px`,
   }))
 
-  if (props.multiple && !isArray(props.modelValue)) {
-    emit(UPDATE_MODEL_EVENT, [])
-  }
-  if (!props.multiple && isArray(props.modelValue)) {
-    emit(UPDATE_MODEL_EVENT, '')
-  }
```

Comment:
Is it a breaking change to remove this code?

Reply:
I think when users set default modelValue, they usually do not pass in an array when selecting a single option or a non array when selecting multiple options. It is a breaking change, but its impact is limited.

Reply:
Maybe we should note this in the documentation or something?